.github/ISSUE_TEMPLATE/feature.yml

```
@@ -0,0 +1,30 @@
+name: 💡 Feature Request
+description: Create a feature request for sentry-python SDK.
```

```suggestion
description: Create a feature request for a sentry-javascript SDK.
```

A few overall thoughts on this template:

1) I think it would be helpful to get SDK name and version here as well, so we can better understand the context in which the request is being made. We might also consider including the "I've already looked to see if this exists" checkbox.

2) Would it make sense here to structure this more like a bug report, with "use case," "current behavior," and "desired behavior" sections? We could also include an optional "suggestions" section at the end.

---

Good ideas.

> I think it would be helpful to get SDK name and version here as well, so we can better understand the context in which the request is being made. We might also consider including the "I've already looked to see if this exists" checkbox.

Sounds good. Let's merge this first and I'll do this later today.

> Would it make sense here to structure this more like a bug report, with "use case," "current behavior," and "desired behavior" sections? We could also include an optional "suggestions" section at the end.

Maybe. 🤷‍♂️  Tbh, I would like this more to follow something like a problem statement or a user story ide more and keep it different from the bug report.